### DISCUSSION TITLE
Consume Eleventy `toNDJSON` stream via `for await`

### DISCUSSION THREAD

**Discussion by monochromer** [11ty/eleventy#2469] [Q&A]
I'm [trying to use programmatic API](https://github.com/monochromer/eleventy-deep-nested-computed-data/blob/json-stream/build.js) for Eleventy and `toNDJSON`.

When i use `.on('data', () => {})` inteface, it works fine - the data flow goes chunk by chunk

```javascript
 stream.on('data', (entry) => {
    consume(entry)
  })
```

But when i use `for await`, all data come in one chunk:

```javascript
for await (const entry of stream) {
    consume(entry)
  }
```

**Comment by zachleat** [comment]
Hmm: this feels like it might be better suited as an issue rather than a discussion!
